AUTHORS

```diff
@@ -0,0 +1,13 @@
+This is a joint attempt of Appsembler and Raccoon Gang to enhance
+Open edX ecosystem. We welcome open-source community to add more
+video-backends as well as fix issues.
+
+Alexander Marenich <alexander.marenich@raccoongang.com>
+Alexander Zayats <alexander.zayats@raccoongang.com>
+Andrey Doroshenko <andrey.doroshenko@raccoongang.com>
+Dmitry Shyshov <dmitry.shishov@raccoongang.com>
+Nate Aune <nate@noderabbit.com>
+Olena Persianova <olena.persianova@raccoongang.com>
+Valera Rozuvan <valera.rozuvan@gmail.com>
+Vladas Tamoshaytis <vladas.tamoshaitis@raccoongang.com>
```

Comment:
Done.
@TamoshaytisV you may also want to check your git config, because I've got the list from the `git log`.